libs/ett/core/shell/src/lib/ett-shell.component.ts

```diff
@@ -35,27 +47,129 @@ const selector = 'ett-shell';
     `
       ${selector} {
         display: block;
+
+        .toolbar {
+          width: 100%;
+          display: flex;
+
+          h1 {
+            flex: auto;
+          }
+
+          .menu {
+            align-self: flex-end;
+          }
+        }
       }
     `,
   ],
   template: `
-    <watt-shell>
+    <div *ngIf="profileLoading">Loading</div>
+
+    <watt-shell *ngIf="!profileLoading">
```

Comment:
issue: the existing test case is failing now because:
- the shell is not displayed until the user has authenticated
- this component now has quite a few concerns, one of them requires an HTTP request, so we need to [use the `HttpClientTestingModule`](https://angular.io/guide/testing-services#testing-http-services) or replace `AuthOidcHttp` with a test double